nservicebus/azure-service-bus/transaction-support.md

```diff
@@ -7,37 +7,66 @@ tags:
 - Transactions
 ---
 
+## Summary
 
-## Transactions and delivery guarantees
+#### Transport transaction - Sends atomic with Receive
 
-NServiceBus Azure Service Bus transport relies on the underlying Azure Service Bus library which requires the use of the `Serializable` isolation level (the most restrictive isolation level that does not permit `dirty reads`, `phantom reads` and `non repeatable reads`; will block any reader until the writer is committed. For more information refer to [Transaction Isolation Levels Explained in Details](http://dotnetspeak.com/2013/04/transaction-isolation-levels-explained-in-details) article.
+Note: `SendAtomicWithReceive` level is supported only as from version 7 and when destination and receive queues are in the same namespace.
 
-NServiceBus Azure Service Bus transport configuration is hard-coded to `Serializable` isolation level. Users can't override it.
+The `SendAtomicWithReceive` guarantee is achieved by using `ViaEntityPath` property on outbound message senders. It's value is set to the receiving queue.
```

Comment:
How about:

The `SendAtomicWithReceive` guarantee is achieved by setting `ViaEntityPath` property on outbound message senders. Its value is set to the receiving queue.